Content.Client/Communications/UI/CommunicationsConsoleMenu.xaml.cs

```diff
@@ -86,9 +107,13 @@ protected override void FrameUpdate(FrameEventArgs args)
         // selected.
         public void UpdateAlertLevels(List<string>? alerts, string currentAlert)
         {
+            // Starlight Start
+            if (AlertLevelCooldownEnd.HasValue)
+                return;
+            // Starlight End
             AlertLevelButton.Clear();
 
-            if (alerts == null)
+            if (alerts == null || !alerts.Any()) // Starlight edit
```

Comment:
```suggestion
            if (alerts?.Any()) // Starlight edit
```

Isn't it's clearlier?